spec/models/vm_or_template_spec.rb

```diff
@@ -471,4 +471,14 @@
       expect(VmOrTemplate.batch_operation_supported?(:migrate, [vm1.id, vm2.id])).to eq(true)
     end
   end
+
+  context ".set_tenant_from_group" do
+    it "assigns the tenant from the group" do
+      tenant = FactoryGirl.create(:tenant, :parent => EvmSpecHelper.create_root_tenant)
+      group = FactoryGirl.create(:miq_group, :tenant => tenant)
+      vm = FactoryGirl.create(:vm_vmware)
+      vm.update_attributes(:miq_group => group)
```

Comment:
Can you assign the miq_group on create?   
`vm = FactoryGirl.create(:vm_vmware, :miq_group => group)`